frontend/src/features/fields/fieldService.tsx

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+const API_URL = '/api/fields/';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const SERVER_URL = process.env.REACT_APP_SERVER_URL || '';
+
+interface FieldServiceObject {
```

Comment:
There are a few different ways to go about this but the data you're passing into the create/edit/delete functions won't actually be in this form. 

For example for editField, we want three values:
- the field name (eg foodType)
- the original value (eg produce)
- the new value (eg fresh produce) 

(got this from field controller line 92 fyi)
So you could create an interface for each call so it matches the structure of the data you pass in like so:
```
interface EditFieldDataObject {
   fieldName: string,
   oldValue: string,
   newValue: string
}
```

(then create and delete could actually share the same interface, with just fieldName and value properties) 
```
interface FieldDataObject {
   fieldName: string,
   value: string
}
```